ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Send.hs

```diff
@@ -154,19 +212,40 @@ initSend uiWalletFace@UiWalletFace{..} sendInputs = do
         eventType <- QEvent.eventType ev
         dropdownVisible <- QWidget.isVisible dropdownWidget
         if eventType == QEvent.Resize && dropdownVisible
-          then moveDropDown dropdownWidget s $> True
+          then
+            moveDropDown dropdownWidget s $> True
           else return False
 
     for_ checkboxes $
       \AccountCheckbox{..} -> connect_ checkbox QAbstractButton.toggledSignal $ \_ -> accountsSelected as s
 
+  void $ Event.onEvent send $
+    \(ev :: QEvent.QEvent) -> do
+      eventType <- QEvent.eventType ev
+      if eventType == QEvent.Resize || eventType == QEvent.Move
```

Comment:
What's going on here?

Reply:
Move add button, when window is being resized or moved

Reply:
Actually, comparison with `QEvent.Move` is unnecessary, cause this button is already sticked to the window